From 7c933f9a521070eec2ee1d76abcef6088935a8c8 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@nabladev.com>
Date: Wed, 12 Aug 2026 21:09:45 +0200
Subject: [PATCH 13639/15333] hwmon: (tmp102) Add TMP110 device ID

The TMP110 is register compatible with TMP102, add non-DT I2C device ID.

Signed-off-by: Marek Vasut <marex@nabladev.com>
Link: https://lore.kernel.org/r/20260812191021.65304-2-marex@nabladev.com
Signed-off-by: Guenter Roeck <linux@roeck-us.net>
---
 drivers/hwmon/tmp102.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/hwmon/tmp102.c b/drivers/hwmon/tmp102.c
index 95fb912a1f7e..26c461e95175 100644
--- a/drivers/hwmon/tmp102.c
+++ b/drivers/hwmon/tmp102.c
@@ -396,6 +396,7 @@ static DEFINE_SIMPLE_DEV_PM_OPS(tmp102_dev_pm_ops, tmp102_suspend, tmp102_resume
 
 static const struct i2c_device_id tmp102_id[] = {
 	{ .name = "tmp102" },
+	{ .name = "tmp110" },
 	{ }
 };
 MODULE_DEVICE_TABLE(i2c, tmp102_id);
-- 
2.55.0

